commit dd040b488836463559515ba244cd3848973e5d6b
Author: Adam Saponara <as@php.net>
Date:   Sat Feb 15 21:23:59 2020 -0500

    maybe fix armel/armhf/hppa debian build fail

diff --git a/util.c b/util.c
index 999c2b2..3aaec30 100644
--- a/util.c
+++ b/util.c
@@ -600,6 +600,7 @@ void str_append_replace_with_backrefs(str_t *str, char *subj, char *repl, int pc
         } else if (*(repl_backref+1) == 'x' && repl_backref+3 < repl_stop) {
             // $xNN; append byte
             strncpy(hex, repl_backref+2, 2);
+            hex[2] = '\0';
             byte = strtoul(hex, NULL, 16);
             term = &byte;
             term_stop = term + 1;
